prov/verbs/src/fi_verbs.c

```diff
@@ -646,7 +658,13 @@ static inline int fi_ibv_get_inject_size(struct ibv_context *ctx,
 		goto err2;
 	}
 
-	info->tx_attr->inject_size = init_attr.cap.max_inline_data;
+	info->tx_attr->inject_size	= init_attr.cap.max_inline_data;
+	info->tx_attr->iov_limit 	= init_attr.cap.max_send_sge;
+	info->tx_attr->rma_iov_limit 	= init_attr.cap.max_send_sge;
```

Comment:
This should be 1.  No devices support more than 1 IOV on the remote side of an operation.